openhtf/core/test_executor.py

```diff
@@ -233,6 +235,14 @@ def _handle_phase(self, phase):
 
     self.test_state.state_logger.debug('Handling phase %s', phase.name)
     outcome = self._phase_exec.execute_phase(phase)
+    if self.test_state.test_options.stop_on_first_failure:
+      # Stop Test on first measurement failure
+      current_phase_result = self.test_state.test_record.phases[
+          len(self.test_state.test_record.phases) -1]
+      if current_phase_result.outcome == openhtf.test_record.PhaseOutcome.FAIL:
+        outcome = phase_executor.PhaseExecutionOutcome(openhtf.PhaseResult.STOP)
```

Comment:
openhtf.PhaseResult.STOP  -> phase_descriptor.PhaseResult.STOP

phase_descriptor will need to be imported

Reply:
Done